src/config/index.ts

```diff
@@ -64,6 +68,12 @@ export function getConfig(): Config {
       .string()
       .default('')
       .transform((val) => (val === '' ? [] : val.split(','))),
+    EARN_SUPPORTED_NETWORK_IDS: z
+      .string()
+      .default('')
+      .transform((val) =>
+        val === '' ? ([] as NetworkId[]) : (val.split(',') as NetworkId[]),
```

Comment:
Is there a reason for the empty string check here but not in prod?

Reply:
here the string can be optional and default to empty string. in prod its not optional so we don't need to handle the empty case